content/news.pml

```diff
@@ -39,6 +39,49 @@
 
 <!----------------------------------------------------------->
 
+<div class="news-item">
+  11 June 2021: seL4 12.1.0 released
+</div>
+<div>
+ <p>
+    <img src="images/seL4.svg" style="width:30%; padding-left:1em; float:right" alt="seL4">
+
+    The seL4 Foundation has released the following updates:
+    <ul>
+      <li>
+        seL4 v12.1.0:
+        <a href="https://github.com/seL4/seL4/releases/tag/12.1.0">release</a>,
+        <a href="https://github.com/seL4/seL4/blob/12.1.0/CHANGES">change log</a>
```

Comment:
The link should go to the more nicely formatted change log on the docs site, I think, i.e.

```suggestion
        <a href="https://docs.sel4.systems/releases/sel4/12.1.0">change log</a>
```